src/mrinufft/trajectories/tools.py

```diff
@@ -700,3 +701,178 @@ def radialize_center(trajectory, nb_samples, in_out=False):
         return _radialize_in_out(trajectory, nb_samples)
     else:
         return _radialize_center_out(trajectory, nb_samples)
+
+
+#################
+# Randomization #
+#################
+
+
+def _flip2center(mask_cols: list[int], center_value: int) -> np.ndarray:
+    """
+    Reorder a list by starting by a center_position and alternating left/right.
+
+    Parameters
+    ----------
+    mask_cols: list or np.array
+        List of columns to reorder.
+    center_pos: int
+        Position of the center column.
+
+    Returns
+    -------
+    np.array: reordered columns.
+    """
+    center_pos = np.argmin(np.abs(np.array(mask_cols) - center_value))
+    mask_cols = list(mask_cols)
+    left = mask_cols[center_pos::-1]
+    right = mask_cols[center_pos + 1 :]
+    new_cols = []
+    while left or right:
+        if left:
+            new_cols.append(left.pop(0))
+        if right:
+            new_cols.append(right.pop(0))
+    return np.array(new_cols)
+
+
+def get_random_loc_1d(
+    dim_size,
+    center_prop,
+    accel=4,
+    pdf="uniform",
+    rng=None,
+    order="center-out",
+) -> np.ndarray:
+    """Get slice index at a random position.
+
+    Parameters
+    ----------
+    dim_size: int
+        Dimension size
+    center_prop: float or int
+        Proportion of center of kspace to continuouly sample
+    accel: float
+        Undersampling/Acceleration factor
+    pdf: str, optional
+        Probability density function for the remaining samples.
+        "gaussian" (default) or "uniform".
+    rng: random state
+
+    Returns
+    -------
+    np.ndarray: array of size dim_size/accel.
+    """
+    order = VDSorder(order)
+    pdf = VDSpdf(pdf)
+    if accel == 0:
+        return np.arange(dim_size)  # type: ignore
+
+    indexes = list(range(dim_size))
```

Comment:
Why pick positions over a discrete grid when uniform and gaussian pdfs are defined over continuous spaces ? We could remove `dim_size` entirely as it would be only relevant when `pdf` is an array which already contains `dim_size` in its shape

Reply:
That's a good point. There are two reason: 1) this function was initially written 2 years ago for creating cartesian mask, and it evolved from there 2) this enforces a stacking that lies on the cartesian grid, making it suitable for Stacked-NUFFT (2.5 D) optimization. 

A solution could be to resort to the continuous case if `dim_size = None` ?